source/api/commands/as.md

```diff
@@ -21,9 +20,9 @@ Assign an alias for later use. Reference the alias later within a {% url `cy.get
 
 ```javascript
 cy.get('.main-nav').find('li').first().as('firstNav') // Alias first 'li' as @firstNav
-cy.route('PUT', 'users', 'fx:user').as('putUser')     // Alias 'route' as @putUser   
-cy.stub(api, 'onUnauth').as('unauth')                 // Alias 'stub' as @unauth   
-cy.spy(win, 'fetch').as('winFetch')                   // Alias 'spy' as @winFetch  
+cy.route('PUT', 'users', 'fx:user').as('putUser')     // Alias that route as @putUser   
+cy.stub(api, 'onUnauth').as('unauth')                 // Alias that stub as @unauth   
+cy.spy(win, 'fetch').as('winFetch')                   // Alias that spy as @winFetch  
```

Comment:
This just read a wee bit oddly to my eyes so I changed it... we're not aliasing each function itself, we're aliasing what is yielded by each functions. That's why I added the _"that"_ qualifier.